README.md

```diff
@@ -1,2 +1,77 @@
-# comp150-agile
-Essay on the Agile Philosophy
+Should scrum be used to teach students for group projects instead of more traditional teaching methods in schools and higher education environments?
+
+Topic:
+
+My essay will be on whether scrum can be used as an effective medium to teach students in primary and higher education. I have chosen to write about this because teaching has changed from the traditional methods of learning to more active learning activities such as having students participate in activities rather than just having them listen to a lesson or lecture. scrum would be an effective way to do this, it can also be applied to lessons very easily. Where in scrum there are customers who can change their mind about a product such as in game development. In a school or university, the lecturer can take over this role to present challenges to pupils. This engages them and helps them to learn.
```

Comment:
There is a really great paper on [active learning](http://www.pnas.org/content/111/23/8410.short) which you could look at. The book by [Hattie](https://books.google.co.uk/books?hl=en&lr=&id=ZO8jmUjQbs0C&oi=fnd&pg=PR5&dq=hattie+visible+learning&ots=1-RwioVVmD&sig=i1U_QY1Ypdt8qdokYp-niM9uQb8#v=onepage&q=hattie%20visible%20learning&f=false).